org.zrythm.Zrythm.json

```diff
@@ -0,0 +1,442 @@
+{
+    "app-id": "org.zrythm.Zrythm",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "41",
+    "sdk": "org.gnome.Sdk",
+    "command": "zrythm_launch",
+    "rename-icon": "zrythm",
+    "copy-icon": true,
+    "finish-args": [
+        "--share=ipc",
+        "--device=dri",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        
+        "--share=network",
+        
+        "--socket=pulseaudio",
+        "--device=all",
+        
```

Comment:
I see no filesystem permission to allow saving files.

Reply:
Some parts of the app don't use GTKFileChooserNative currently, so indeed some filesystem permission is needed, I added filesystem=home. There is also a file browser within the app, so that might not be possible to "fix" without granting lots of permissions.